test/benchmark_http_client_test.cc

```diff
@@ -26,6 +26,27 @@ using namespace testing;
 
 namespace Nighthawk {
 
+namespace {
+const std::shared_ptr<Envoy::Http::RequestHeaderMap>& getDefaultHeaderMap() {
+  static auto* header =
+      new std::shared_ptr<Envoy::Http::RequestHeaderMap>{new Envoy::Http::TestRequestHeaderMapImpl{
+          {":scheme", "http"}, {":method", "GET"}, {":path", "/"}, {":host", "localhost"}}};
+  return *header;
+}
+
+// Default function for request generator when the content doesn't matter.
+RequestGenerator getDefaultRequestGenerator() {
+  RequestGenerator request_generator = []() {
+    return std::make_unique<RequestImpl>(getDefaultHeaderMap());
+  };
+  return request_generator;
+}
+// Helper function to get headers in a set that should be verified during the test.
+std::string getTestRecordedProperty(const Envoy::Http::RequestHeaderMap& header) {
```

Comment:
Perhaps consider updating this to `getPathFromRequest()` and update the comment above, to match  it to the narrowed down functionality?

Reply:
done.